app/controllers/projects/samples_controller.rb

```diff
@@ -54,16 +54,15 @@ def update
       end
     end
 
-    def destroy
+    def destroy # rubocop:disable Metrics/AbcSize
       ::Samples::DestroyService.new(@sample, current_user).execute
       @pagy, @samples = pagy(Sample.where(project_id: @project.id))
-      respond_to do |format|
-        if @sample.deleted?
-          format.turbo_stream do
-            render status: :ok, locals: { type: 'success',
-                                          message: t('.success', sample_name: @sample.name) }
-          end
-        else
+
+      if @sample.deleted?
+        flash[:success] = t('.success', sample_name: @sample.name, project_name: @project.name)
```

Comment:
For the project_name variable we should get the name with the path since many different projects can have the same name but under different namespaces. Let's update this to `project_name: @project.namespace.human_name`

Reply:
That makes sense! Fixed in 04b9ef73d5d5cb7e8ed5502c7e7c41b152494d4e